src/main/java/tests/FileReaderWriterTest.java

```diff
@@ -0,0 +1,29 @@
+package tests;
+
+import core.basesyntax.FileReader;
+import core.basesyntax.FileReaderImpl;
+import core.basesyntax.FileWriter;
+import core.basesyntax.FileWriterImpl;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class FileReaderWriterTest {
+    @Test
+    public void testFileReaderWriter() throws IOException {
+        String filePath = "testReport.csv";
+        String content = "fruit,quantity\nbanana,152\napple,90\n";
+        FileWriter fileWriter = new FileWriterImpl();
+        fileWriter.write(content, filePath);
+        FileReader fileReader = new FileReaderImpl();
+        List<String> readContent = fileReader.read(filePath);
+        assertEquals(Arrays.asList(content.split("\n")), readContent);
+        Files.delete(Paths.get(filePath));
+    }
+}
```

Comment:
The test class should be placed in the corresponding package that reflects the package structure of the classes it tests. Since `FileReader` and `FileWriter` are in the `core.basesyntax` package, consider placing this test class in a package like `core.basesyntax.tests` to maintain a clear structure.